src/content/en/updates/2018/12/badging-api.md

```diff
@@ -81,6 +81,11 @@ Examples of sites that may use this API include:
    running as an installed PWA (in your task bar or dock).
 3. Click the **Set** or **Clear** button to set or clear the badge from the app
    icon. You can also provide a number for the *Badge value*.
+   
+Note: While the Badging API requires an installed app with an icon that
```

Comment:
I think even this is maybe giving the wrong impression: the badging API doesn't necessarily only apply to the installed app. That's simply currently how it is in Chrome. The badging API can apply to anywhere the browser might want to show a badge. So developers shouldn't make any assumptions about what situations the browser will make badges work. Just call the API if it exists.

Reply:
Thanks! I hope this feedback is dealt with in https://github.com/google/WebFundamentals/pull/7999/commits/9ec4212a74fbc2a285e9d162e1acada2852ed6e5.